back/db/migrate/20210618161354_add_inputs_count_to_category.rb

```diff
@@ -0,0 +1,5 @@
+class AddInputsCountToCategory < ActiveRecord::Migration[6.0]
+  def change
+  #  add_column :insights_categories, :inputs_count, :integer, null: false, default: 0
```

Comment:
I have a hacky soluion for my hacky fix to the weird mistake, I comment the content of the migration, 
  deploy (so run the migrations, which will add this date to the migrations without doing anything),
  then uncomment this so you can replay history.
  On my local it works, how dumb is it ? Should I just contemplate my mistake untill 2022 instead ? Any other ideas ?